### PR TITLE
Do not include TZ information in queries

### PR DESCRIPTION
**Pull request by walro**
Same as https://github.com/twingly/twingly-search-api-ruby/pull/69.

Re-record fixtures to get fixtures without published time bugs.

### CODE REVIEW

File: tests/fixtures/vcr_cassettes/search_without_valid_api_key.json [hunk 1]
```diff
@@ -1,43 +1 @@
...
-        },
-        "url": "https://api.twingly.com/blog/search/api/v3/search?q=something&apiKey=wrong"
-      },
-      "recorded_at": "2017-05-13T20:29:14"
-    }
-  ],
-  "recorded_with": "betamax/0.8.0"
-}
\ No newline at end of file
+{"http_interactions": [{"request": {"body": {"string": "", "encoding": "utf-8"}, "headers": {"Connection": "keep-alive", "Accept-Encoding": "gzip, deflate", "Accept": "*/*", "User-Agent": "Twingly Search Python Client/2.1.0"}, "method": "GET", "uri": "https://api.twingly.com/blog/search/api/v3/search?q=something&apiKey=wrong"}, "response": {"body": {"string": "<?xml version=\"1.0\" encoding=\"utf-8\"?><error code=\"40101\"><message>Unauthorized</message></error>", "encoding": "utf-8"}, "headers": {"Content-Length": "97", "Expires": "-1", "Server": "nginx", "Connection": "keep-alive", "Pragma": "no-cache", "Cache-Control": "no-cache", "Date": "Mon, 16 Oct 2017 08:54:09 GMT", "Content-Type": "application/xml; charset=utf-8"}, "status": {"message": "Unauthorized", "code": 401}, "url": "https://api.twingly.com/blog/search/api/v3/search?q=something&apiKey=wrong"}, "recorded_at": "2017-10-16T08:54:09"}], "recorded_with": "betamax/0.5.1"}
```
Comment (walro) [inline tests/fixtures/vcr_cassettes/search_without_valid_api_key.json]: hmm I should probably update my betamax so get nicer output :) *cough* #11

Comment (walro) [reply tests/fixtures/vcr_cassettes/search_without_valid_api_key.json]: didn't matter, maybe it's Windows vs macOS thing.

File: tests/test_query.py [hunk 2]
```diff
@@ -58,14 +58,14 @@ def test_query_should_add_start_date(self):
         q.search_query = "spotify"
         q.start_time = self.datetime_with_timezone(datetime(2012, 12, 28, 9, 1, 22), "UTC")
         query_string = q.build_query_string()
-        self.assertEqual(query_string, "spotify start-date:2012-12-28T09:01:22Z")
+        self.assertEqual(query_string, "spotify start-date:2012-12-28 09:01:22")
```
Comment (roback) [inline tests/test_query.py]: Shouldn't it be `"spotify start-date:\"2012-12-28 09:01:22\""` (surrounded by quotes) now that the timestamp contains a space?

Comment (walro) [reply tests/test_query.py]: I don't think, the request sent will actually include %20 (or +).  Also take a look at what we had before: 4e4ef2bf116deb2b33dd473e658f16b228833e96

Comment (walro) [reply tests/test_query.py]: But yeah, I see our docs includes quoting, but I don't think it's necessary.

Comment (walro) [reply tests/test_query.py]: Hmm, investigated some more. Seems like you are right. We probably have this bug in most of clients. Nice catch.

Comment (roback) [reply tests/test_query.py]: Adding a `T` between the date and the time parts works as well.

Comment (walro) [reply tests/test_query.py]: > Adding a T between the date and the time parts works as well.

Yes, seems to be working. I am thinking we should possibly update our developer site to prefer this over quoting. Should be easier for customers and also easier for us when reading logs.

**Review by roback — APPROVED** [review]

**Pull request merged**